### PR TITLE
Admin units API from beta to v1

### PR DESCRIPTION
**Pull request by anandyadavMSFT**
Creating pull request for migrating AU APIs from beta to V1

### CODE REVIEW

**Comment by anandyadavMSFT** [comment]
@davidmu1 : Can you please help with the error it is showing for microsoft-graph-docs-apidoctor? I am not sure I understand why is it failing now or where to see the error details so that I can fix it.

**Comment by anandyadavMSFT** [comment]
@davidmu1 : Can you please help?

File: api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "delete_scopedrolemember"
+}-->
+```http
+DELETE https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}/scopedRoleMembers/{id}
+```
+# [C#](#tab/csharp)
+[!INCLUDE [sample-code](../includes/snippets/csharp/delete-scopedrolemember-csharp-snippets.md)]
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md:57]: The following comments are generally applicable to the errors you're currently getting about some error in an include file not found, like this:
"[Error-include-not-found] Invalid include link: '../includes/snippets/csharp/delete-scopedrolemember-csharp-snippets.md'."

Using the following few lines 57, 61, 65 that have that error as examples:
When you're creating a new API topic, or copying a beta topic and adapting it to v1, please exclude these lines that attempt to insert a snippet file into the new topic. As a matter of fact, in this topic, you can just delete lines 56 through 68.

These snippet files don't yet exist in the folder you specified and hence you're getting build errors. Once your PR is eventually merged to master, the snippet generator which runs weekly would detect that this is a new file and would generate and insert the INCLUDE lines to incorporate snippets for your examples.

File: api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Response
+
+If successful, this method returns `204 No Content` response code. It does not return anything in the response body.
+
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md:48]: Remove "# [HTTP](#tab/http)"

File: api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "delete_scopedrolemember"
+}-->
+```http
+DELETE https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}/scopedRoleMembers/{id}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md:56]: Remove lines 56 through 68.

File: api-reference/v1.0/api/administrativeunit-delete.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+## Response
+
+If successful, this method returns `204 No Content` response code. It does not return anything in the response body.
+
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delete.md:48]: Remove line 48.

File: api-reference/v1.0/api/administrativeunit-delete.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "delete_administrativeunit"
+}-->
+```http
+DELETE https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delete.md:56]: Remove lines 56 through 68.

File: api-reference/v1.0/api/administrativeunit-delta.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+- If a `deltaLink` URL is returned, there is no more data about the existing state of the resource to be returned. Persist and use the `deltaLink` URL to learn about changes to the resource in the future.
+
+For details and an example, see [Using delta query](/graph/delta-query-overview) and [Get incremental changes for users](/graph/delta-query-users).
+
+## Example
+### Request
+
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delta.md:84]: Remove line 84.

File: api-reference/v1.0/api/administrativeunit-delta.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "administrativeunit_delta"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/administrativeunits/delta
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-delta.md:92]: Remove lines 92 though 104.

File: api-reference/v1.0/api/administrativeunit-get-scopedrolemembers.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and the requested [scopedRoleMembership](../resources/scopedrolemembership.md) object in the response body.
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-get-scopedrolemembers.md:48]: Remove line 48.

File: api-reference/v1.0/api/administrativeunit-get-scopedrolemembers.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_scopedrolemember"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}/scopedRoleMembers/{id}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-get-scopedrolemembers.md:56]: Remove lines 56 through 68.

File: api-reference/v1.0/api/administrativeunit-get.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and [administrativeUnit](../resources/administrativeunit.md) object in the response body.
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-get.md:51]: Remove line 51.

File: api-reference/v1.0/api/administrativeunit-get.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_administrativeunit"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-get.md:59]: Remove lines 59 through 71.

File: api-reference/v1.0/api/administrativeunit-list-scopedrolemembers.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and collection of [scopedRoleMembership](../resources/scopedrolemembership.md) objects in the response body.
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-list-scopedrolemembers.md:48]: Remove line 48.

File: api-reference/v1.0/api/administrativeunit-list-scopedrolemembers.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_scopedrolemember"
+}-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/directory/administrativeUnits/{id}/scopedRoleMembers
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-list-scopedrolemembers.md:56]: Remove lines 56 through 68.

File: api-reference/v1.0/api/administrativeunit-list.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and collection of [administrativeUnit](../resources/administrativeunit.md) objects in the response body.
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-list.md:48]: Remove line 48.

File: api-reference/v1.0/api/administrativeunit-post-administrativeunits.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+
+If successful, this method returns a `201 Created` response code and an [administrativeUnit](../resources/administrativeunit.md) object in the response body.
+
+## Example
+
+### Request
+
+The following is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-post-administrativeunits.md:52]: Remove line 52.

File: api-reference/v1.0/api/administrativeunit-post-administrativeunits.md [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+Content-type: application/json
+Content-length: 150
+
+{
+    "displayName": "Seattle District Technical Schools",
+    "description": "Seattle district technical schools administration",
+    "visibility": "HiddenMembership"
+}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-post-administrativeunits.md:68]: Remove lines 68 through 80.

File: api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+
+## Response
+
+If successful, this method returns `201 Created` response code and [scopedRoleMembership](../resources/scopedrolemembership.md) object in the response body.
+
+## Example
+##### Request
+Here is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md:47]: Remove line 47.

File: api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+Content-length: 272
+
+{
+  "roleId": "roleId-value",
+  "roleMemberInfo": {
+    "id": "id-value"
+  }
+}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md:64]: Remove lines 64 through 76.

File: api-reference/v1.0/api/administrativeunit-update.md [hunk 19]
```diff
@@ -0,0 +1,122 @@
...
+## Response
+
+If successful, this method returns a `204 No Content` response code.
+
+## Example
+
+##### Request
+
+
+# [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-update.md:59]: Remove line 59.

File: api-reference/v1.0/api/administrativeunit-update.md [hunk 20]
```diff
@@ -0,0 +1,122 @@
...
+Content-type: application/json
+Content-length: 114
+
+{
+  "displayName": "displayName-value",
+  "description": "description-value",
+  "visibility": "visibility-value"
+}
+```
+# [C#](#tab/csharp)
```
Comment (davidmu1) [inline api-reference/v1.0/api/administrativeunit-update.md:75]: Remove lines 75 through 87.

**Pull request merged**